app/.eslintrc

```diff
@@ -20,6 +20,7 @@
     "jsx-a11y/no-static-element-interactions": 0,
     "global-require": 0,
     "linebreak-style": 0,
+    "no-bitwise": 0,
```

Comment:
Let's not use bitwise since they make code less readable and also may lead to unexpected behavior

Reply:
fixed

Reply:
fixed